<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.985000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.115385 rgba(130, 0, 122, 241), stop:1 rgba(90, 129, 255, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>971</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
	background:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.115385 rgba(0, 130, 113, 241), stop:1 rgba(114, 83, 224, 248));
	color: rgb(240, 240, 240);
	opacity: 0.6;
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>70</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QWidget" name="filterMarks_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="filterMarks">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string> Range:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>From</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="fromMark">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 162, 106, 172), stop:1 rgba(124, 157, 255, 197));
	border: 1px solid rgb(115, 128, 158);
	height: 30px;
	border-radius: 0px;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>To</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="toMark">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 162, 106, 172), stop:1 rgba(124, 157, 255, 197));
	border: 1px solid rgb(115, 128, 158);
	height: 30px;
	border-radius: 0px;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="filterMarksB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 10px;
	height: 50px;
	background: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.115385 rgba(18, 100, 121, 238), stop:1 rgba(204, 50, 78, 198));
	color: rgb(255, 255, 255);
}

QPushButton:pressed{
	background-color:qconicalgradient(cx:0.5, cy:0.5, angle:43.5, stop:0.114428 rgba(25, 101, 121, 203), stop:1 rgba(204, 50, 78, 190));
}</string>
       </property>
       <property name="text">
        <string>Filter by marks</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="filterDate_2" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="filterDate">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>   Year:     </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yearToShow">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 162, 106, 172), stop:1 rgba(124, 157, 255, 197));
	border: 1px solid rgb(212, 197, 255);
	border-radius: 5px;
	height: 30px;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>   Month:   </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="monthToShow">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 162, 106, 172), stop:1 rgba(124, 157, 255, 197));
	border: 1px solid rgb(212, 197, 255);
	border-radius: 5px;
	height: 30px;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="filterDateB">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 10px;
	height: 50px;
	background: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.115385 rgba(18, 100, 121, 238), stop:1 rgba(204, 50, 78, 198));
	color: rgb(255, 255, 255);
}

QPushButton:pressed{
	background-color:qconicalgradient(cx:0.5, cy:0.5, angle:43.5, stop:0.114428 rgba(25, 101, 121, 203), stop:1 rgba(204, 50, 78, 190));
}</string>
       </property>
       <property name="text">
        <string>Filter by date</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="StudentDel_2" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="StudentDel">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>      First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fNameToDel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 162, 106, 172), stop:1 rgba(124, 157, 255, 197));
	border: 1px solid rgb(212, 197, 255);
	border-radius: 5px;
	height: 30px;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>      Last Name </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lNameTodel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 162, 106, 172), stop:1 rgba(124, 157, 255, 197));
	border: 1px solid rgb(212, 197, 255);
	border-radius: 5px;
	height: 30px;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="delMember">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 10px;
	height: 50px;
	background: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.115385 rgba(18, 100, 121, 238), stop:1 rgba(204, 50, 78, 198));
	color: rgb(255, 255, 255);
}

QPushButton:pressed{
	background-color:qconicalgradient(cx:0.5, cy:0.5, angle:43.5, stop:0.114428 rgba(25, 101, 121, 203), stop:1 rgba(204, 50, 78, 190));
}</string>
       </property>
       <property name="text">
        <string>Delete from list</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="resetB">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid black;
	border-radius: 10px;
	height: 50px;
	background: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.115385 rgba(18, 100, 121, 238), stop:1 rgba(204, 50, 78, 198));
	color: rgb(255, 255, 255);
}

QPushButton:pressed{
	background-color:qconicalgradient(cx:0.5, cy:0.5, angle:43.5, stop:0.114428 rgba(25, 101, 121, 203), stop:1 rgba(204, 50, 78, 190));
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuList">
    <property name="title">
     <string>List</string>
    </property>
    <addaction name="actionAdd_New_Member"/>
   </widget>
   <addaction name="menuList"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_New_Member">
   <property name="text">
    <string>Add New Member</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
